records_mover/records/schema/field/__init__.py

```diff
@@ -185,8 +185,13 @@ def cast_series_type(self, series: 'Series') -> 'Series':
                 # goofy - 1pm will come out as: "0 days 01:00:00".
                 #
                 if type(series[0]) == pd.Timedelta:
+                    # https://stackoverflow.com/questions/34501930/how-to-convert-timedelta-to-time-of-day-in-pandas
+
                     # Convert from "0 days 12:34:56.000000000" to "12:34:56"
-                    return series.astype(str).str.split().str[-1].str.split('.').str[0]
+                    def components_to_time_str(df: pd.DataFrame) -> str:
+                        return f"{df['hours']:02d}:{df['minutes']:02d}:{df['seconds']:02d}"
+                    out = series.dt.components.apply(axis=1, func=components_to_time_str)
+                    return out
```

Comment:
I don't know how much more or less efficient this will be.  It's probably not hugely important until we start dealing with very large things exported from MySQL - and given we don't currently have a bulk unloader, adding one is probably a higher priority than trying to optimize this.  🤷

Reply:
Word-- I think my expectation is that before we worry about this time transformation in pandas being a bottleneck for big datasets, we probably need to worry about pandas itself being the bottleneck.